### PR TITLE
"done by subrat"

### PR DESCRIPTION
**Pull request by Subrat29**
1. add MemberPosition in TeamMember.tsx and use it in Team.tsx
2. next/static/css/app/Layout.css --> .flex-wrap: justifycontent: center

### CODE REVIEW

File: components/team/Team.tsx [hunk 1]
```diff
@@ -4,7 +4,11 @@ import TeamMember from './TeamMember';
 const Team: React.FC = () => {
     const teamDetails = [
         {
-            name: "Akansha",
+            name: "Akansha", 
+
+            // add position by subrat
+            MemberPosition: "(Team Member)",
+
```
Comment (anurag629) [inline components/team/Team.tsx]: remove the extra lines and comment. Also, remove the curly braces

File: components/team/Team.tsx [hunk 2]
```diff
@@ -16,6 +20,10 @@ const Team: React.FC = () => {
         },
         {
             name: "Rukshali",
+
```
Comment (anurag629) [inline components/team/Team.tsx]: same here extra lines and comments

File: components/team/TeamMember.tsx [hunk 3]
```diff
@@ -50,6 +56,10 @@ const TeamMember: React.FC<TeamMemberProps> = ({
                     <div className="text-center mt-6">
                         <h1 className="text-white-900 text-xl font-bold mb-1">{name}</h1>
                         <div className="text-red-500 font-light mb-2">{title}</div>
+
```
Comment (anurag629) [inline components/team/TeamMember.tsx]: remove every extra lines and comments

**Review by anurag629 — CHANGES_REQUESTED** [review]
You didn't add the code to the center of the image

File: components/team/Team.tsx (hunk 2 above)
Comment (Subrat29) [reply components/team/Team.tsx]: okey

File: components/team/Team.tsx (hunk 1 above)
Comment (Subrat29) [reply components/team/Team.tsx]: okey

File: components/team/TeamMember.tsx (hunk 3 above)
Comment (Subrat29) [reply components/team/TeamMember.tsx]: okey

**Review by anurag629 — APPROVED** [review]

**Comment by anurag629** [comment]
lgtm

**Pull request merged**